composer.json

```diff
@@ -1,109 +1,150 @@
+    "doctrine/doctrine-migrations-bundle": "^1.0",
+    "doctrine/orm": "~2.5.0",
+    "guzzlehttp/guzzle": "^6.3",
+    "incenteev/composer-parameter-handler": "^2.1",
+    "jeremykendall/php-domain-parser": "~1.3.1",
+    "knplabs/knp-menu-bundle": "^2.1",
+    "league/tactician-bundle": "^0.4.1",
+    "lesstif/php-jira-rest-client": "^1.33",
+    "lexik/translation-bundle": "^4.0",
+    "nelmio/security-bundle": "^2.9",
+    "openconext/monitor-bundle": "^1.0",
+    "ramsey/uuid": "^3.7",
+    "sensio/distribution-bundle": "^5.0",
+    "sensio/framework-extra-bundle": "^3.0",
+    "stfalcon/tinymce-bundle": "^2.1",
+    "stof/doctrine-extensions-bundle": "^1.2",
+    "surfnet/stepup-saml-bundle": "^4.0",
+    "symfony/monolog-bundle": "^3.1.0",
+    "symfony/polyfill-apcu": "^1.0",
+    "symfony/swiftmailer-bundle": "^2.3.10",
+    "symfony/symfony": "3.4.*",
+    "twig/twig": "^1.34.4",
+    "xemlock/htmlpurifier-html5": "^0.1.10"
+  },
+  "require-dev": {
+    "doctrine/doctrine-fixtures-bundle": "^2.4",
+    "jakub-onderka/php-parallel-lint": "^0.9.2",
+    "malukenho/docheader": "^0.1.6",
+    "mockery/mockery": "^1.2",
+    "phpmd/phpmd": "~2.8.0",
+    "phpunit/phpcov": "~5.0",
+    "phpunit/phpunit": "~7.5",
+    "sebastian/phpcpd": "^4.1",
+    "sensio/generator-bundle": "^3.0",
+    "squizlabs/php_codesniffer": "^2.8",
+    "symfony/phpunit-bridge": "^3.0"
+  },
+  "scripts": {
+    "symfony-scripts": [
+      "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
+      "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
+      "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
+      "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile"
+    ],
+    "post-install-cmd": [
+      "@symfony-scripts"
+    ],
+    "post-update-cmd": [
+      "@symfony-scripts"
+    ],
+    "check": [
+      "@phpcs",
+      "@phpcs_tests",
+      "@phpmd",
+      "@docheader",
+      "@parallel-lint",
+      "@yaml-lint",
+      "@ts-lint",
+      "@style-lint",
+      "@cvalidate",
+      "@attrsc",
+      "@attrdc",
+      "@droptestdb",
+      "@createtestdb",
+      "@test"
+    ],
+    "test-db": [
+      "@droptestdb",
+      "@createtestdb"
+    ],
+    "disabledebug": "./ci/qa/disabledebug",
+    "jscpd": "./ci/qa/jscpd",
+    "phpcs": "./ci/qa/phpcs",
+    "phpcs_tests": "./ci/qa/phpcs_tests",
+    "phpmd": "./ci/qa/phpmd",
+    "docheader": "./ci/qa/docheader",
+    "parallel-lint": "./ci/qa/parallel-lint",
+    "yaml-lint": "./ci/qa/yaml-lint",
+    "ts-lint": "./ci/qa/ts-lint",
+    "style-lint": "./ci/qa/style-lint",
+    "cvalidate": "./ci/qa/cvalidate",
+    "attrdc": "./ci/qa/attrdc",
+    "attrsc": "./ci/qa/attrsc",
+    "droptestdb": "./ci/qa/droptestdb",
+    "createtestdb": "./ci/qa/createtestdb",
+    "phpcov": "./ci/qa/phpcov",
+    "test": "./ci/qa/phpunit",
+    "cypress": "./ci/qa/cypress",
+    "yarn-audit": "./ci/qa/yarn-audit",
```

Comment:
Suggestion: rename the yarn-audit script (and Composer script) to security-check (or another cool name) and run both `yarn audit` and  `compser audit` in one go.

Reply:
Yes.. simple security might be a good candidate